stock_account_deposit/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
```

Comment:
I think that would be better in all files:

Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>

but is a minor change.